src/styles/components/_Header.scss

```diff
@@ -1,5 +1,7 @@
 // stylelint-disable declaration-property-unit-whitelist
 .header {
+    transition: all 0.3s ease-in-out;
```

Comment:
The nav bar is sticky on all of the mobile devices I tested using Chrome. Due to the background color's transition time, a user can briefly see page content beneath the navbar if they are scrolling quickly. What do you think of this?

Because the effect is so brief, I don't think it is a blocking issue. I tried shortening the transition time to `0.1s`, but it persisted. We might circumvent this with a solid background color across the entire site, although the transition is more visually interesting, especially on pages like `/use-cases`!

<img width="613" alt="Screen Shot 2022-07-01 at 10 00 04 AM" src="https://user-images.githubusercontent.com/60713139/176919730-8ef262e1-b87b-49c1-8f2c-ba83dc0f265a.png">

Reply:
It's a trade off I accepted considering this will only happen for scroll-happy users haha. 😝 It's subtle, but while we have page templates like the `use-case/*` pages which include the nav within the hero, I think we can apply `bg-white` to the nav parent element to avoid this. Another solution I had was to negatively position the nav above the viewport and have it fade down into view after scrolling down, but maybe this can be a secondary approach or something we can suggest during a future redesign.

Reply:
Thank you for walking through the options! Changing it to `bg-white` as you did to the parent in 21dd476 sounds good.